src/main/java/com/crud/cadastrousuario/domain/repository/BookRepositorySpec.java

```diff
@@ -0,0 +1,44 @@
+package com.crud.cadastrousuario.domain.repository;
+
+import com.crud.cadastrousuario.domain.dto.BookDTO;
+import com.crud.cadastrousuario.domain.dto.UserDTO;
+import com.crud.cadastrousuario.domain.model.Book;
+import com.crud.cadastrousuario.domain.model.User;
+import jakarta.persistence.criteria.CriteriaBuilder;
+import org.springframework.data.jpa.domain.Specification;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class BookRepositorySpec {
+
+
+
+    public static Specification<Book> filter(BookDTO filter) {
+        return Specification
+                .where(filterWhereIn("title", filter.getTitle()))
```

Comment:
o que quer dizer o In no nome do metodo?

Reply:
Eu entendo o FilterWhereIn como filtrar onde está